apps/io-profile/src/types/email.ts

```diff
@@ -0,0 +1,8 @@
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { HtmlToTextOptions } from "html-to-text";
+
+export type EmailDefaults = {
+  from: NonEmptyString;
+  htmlToTextOptions: HtmlToTextOptions;
+  title: string;
```

Comment:
Can we define `title` as `NonEmptyString` as well?

Reply:
done in https://github.com/pagopa/io-auth-n-identity-domain/pull/256/commits/d3f64f655db0a101c94c85b50bd4ade63d19e896